client/src/router.js

```diff
@@ -8,10 +8,16 @@ import QuickSearchPage from '../src/pages/QuickSearchPage.vue';
 import ResetPassword from './pages/ResetPassword.vue';
 import SearchResultPage from '../src/pages/SearchResultPage.vue';
 
-export const PRIVATE_ROUTES = ['/change-password'];
+import acronym from 'pdap-design-system/images/acronym.svg';
 
 const routes = [
-	{ path: '/', component: QuickSearchPage, name: 'QuickSearchPage' },
+	{
+		path: '/',
+		component: QuickSearchPage,
+		name: 'QuickSearchPage',
+		// Use meta property on route to override meta tag defaults
+		// meta: { title: 'Police Data Accessibility Project - Search', metaTags: [{ property: 'og:title', title: 'Police Data Accessibility Project - Search' }] },
```

Comment:
Hey @josh-chamberlain  just wanted to call your attention to this. We can override any meta tag by route (in absence of specific route data, we fall back to the defaults). So if there are any of them that should have specific values, let me know and I can add those here as well.